docker/docker-compose.yml

```diff
@@ -138,4 +152,21 @@ services:
       - 8025:8025
       - 1025:1025
 
+  # local-kms is a local KMS-compatible key management service, to replace AWS KMS in local development
+  local-kms:
+    image: nsmithuk/local-kms:3.11.7
+    ports:
+      - 4566:4566
+    environment:
+      PORT: 4566
+      KMS_REGION: eu-central-1
+    volumes:
+      - ./local-kms-seed/seed.yaml:/init/seed.yaml
   
+  jwks:
```

Comment:
Could you add a comment also for this saying it's an HTTP server that serves the local well-known JWKS JSON or similar?